### PR TITLE
fix mypy version < to <=

### LINKED ISSUE #486: No compatibility with the new version of mypy
# Bug report

## What's wrong

After updating mypy to the newest version I got this Error message:
```console
django-stubs 1.6.0 requires mypy<0.790,>=0.782, but you'll have mypy 0.790 which is incompatible.
```

## How is that should be

could you please update django-stubs so it can be compatible with the new version of mypy, thank you

## System information

- OS: ubuntu 20.04
- `python` version: 3.8
- `django` version: 3.1.2
- `mypy` version: 0.790
- `django-stubs` version: 1.6.0

### PR DESCRIPTION
**Pull request by kszmigiel**
closes #486

### CODE REVIEW

**Comment by kszmigiel** [comment]
@sobolevn So 1.6.1 it is 😂

File: setup.py [hunk 1]
```diff
@@ -21,9 +21,9 @@ def find_stub_files(name: str) -> List[str]:
     readme = f.read()
 
 dependencies = [
-    "mypy>=0.782,<0.790",
-    "typing-extensions",
-    "django",
+    'mypy>=0.782,<=0.790',
```
Comment (sobolevn) [inline setup.py]: I guess it should be `'mypy>=0.782` 🤔

File: setup.py [hunk 2]
```diff
@@ -21,9 +21,9 @@ def find_stub_files(name: str) -> List[str]:
...
+    'typing-extensions',
+    'django',
```
Comment (sobolevn) [inline setup.py:26]: Btw, let's use `black` on `setup.py` as well.

File: django-stubs/test/runner.pyi [hunk 3]
```diff
@@ -70,7 +70,7 @@ class ParallelTestSuite(TestSuite):
     processes: Any = ...
     failfast: Any = ...
     def __init__(self, suite: Any, processes: Any, failfast: bool = ...) -> None: ...
-    def run(self, result: Any): ...
+    def run(self, result: Any, debug: Optional[bool]) -> Any: ...
```
Comment (sobolevn) [inline django-stubs/test/runner.pyi]: Does `debug` have a default? If so, we need to specify that.

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks a lot!

File: django-stubs/test/runner.pyi [hunk 4]
```diff
@@ -70,7 +70,7 @@ class ParallelTestSuite(TestSuite):
     processes: Any = ...
     failfast: Any = ...
     def __init__(self, suite: Any, processes: Any, failfast: bool = ...) -> None: ...
-    def run(self, result: Any, debug: Optional[bool]) -> Any: ...
+    def run(self, result: Any, debug: Optional[bool] = False) -> Any: ...
```
Comment (sobolevn) [inline django-stubs/test/runner.pyi:73]: `= ...`

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks! The last nitpick!

**Comment by Goldziher** [comment]
There is already a PR that is doing this, https://github.com/typeddjango/django-stubs/pull/485

also the changes to the test runner are incorrect here - its an override.

File: .travis.yml [hunk 5]
```diff
@@ -38,7 +38,7 @@ jobs:
 
     - name: Lint with black
       python: 3.7
-      script: 'black --check django-stubs/'
+      script: 'black --check django-stubs/ setupy.py'
```
Comment (sobolevn) [inline .travis.yml:41]: `setup.py`

**Review by sobolevn — CHANGES_REQUESTED** [review]

**Comment by sobolevn** [comment]
Fixed in #501

File: setup.py (hunk 2 above)
Comment (oladhari) [reply setup.py:26]: @kszmigiel and @sobolevn sorry for commenting on merged PR, does this means that it supports any new version of mypy, now mypy has updated version to `0.812` in the table in Readme file:https://github.com/typeddjango/django-stubs#version-compatibility
django-stubs supports till mypy version 0.790
thank you

**Pull request closed without merging**